h-raylib.cabal

```diff
@@ -58,10 +63,13 @@ library
   default-language:   Haskell2010
 
   if flag(platform-windows) || (flag(detect-platform) && os(windows))
-    extra-libraries:  opengl32 gdi32 winmm shell32
+    if flag(mingw-cross)
+      extra-libraries:  opengl32 gdi32 winmm shell32 gcc
+    else
+      extra-libraries:  opengl32 gdi32 winmm shell32
     cc-options:       -DPLATFORM_DESKTOP
   elif flag(platform-linux) || (flag(detect-platform) && os(linux))
-    extra-libraries:  GL c m pthread dl rt X11 Xinerama Xcursor Xrandr Xi
+    extra-libraries:  GL c m pthread dl rt X11 Xinerama Xcursor Xrandr Xi Xext
```

Comment:
`X11/extensions/Xge.h` was not found without it listed here. Seems like raylib does have a dependency on `Xext` given that include.